### PR TITLE
LZMA update for 2.0.4

### PR DESCRIPTION
**Pull request by JunTaoLuo**
I think we also need to update the config.dotnet.runtime.version parameter on the CI when we run.

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -202,18 +202,18 @@
     <ExternalDependency Include="Microsoft.Web.Xdt" Version="1.4.0" Source="$(DefaultNuGetFeed)"/>
     <ExternalDependency Include="Microsoft.Win32.Registry" Version="4.4.0" Source="$(DefaultNuGetFeed)" />
     <ExternalDependency Include="Moq" Version="4.7.49" Source="$(DefaultNuGetFeed)" Private="true"/>
-    <ExternalDependency Include="NETStandard.Library" Version="2.0.0" Source="$(DefaultNuGetFeed)">
+    <ExternalDependency Include="NETStandard.Library" Version="2.0.1" Source="$(DefaultNuGetFeed)">
```
Comment (JunTaoLuo) [inline build/dependencies.props]: Not sure if these are needed anymore.

**Comment by JunTaoLuo** [comment]
So I should commit these changes to release/2.0.0 @natemcmaster ?

File: build/dependencies.props (hunk 1 above)
Comment (natemcmaster) [reply build/dependencies.props]: They are for the 2.0.x patch builds, at least for now.

File: build/tools/templates/Archive/Archive.csproj [hunk 2]
```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <EnableApiCheck>false</EnableApiCheck>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
```
Comment (natemcmaster) [inline build/tools/templates/Archive/Archive.csproj]: +💯. Thanks for adding this

**Comment by natemcmaster** [comment]
Yes, this should merge to release/2.0.0

File: build/tools/templates/Archive/Archive.csproj (hunk 2 above)
Comment (JunTaoLuo) [reply build/tools/templates/Archive/Archive.csproj]: I think we should investigate whether we want to have this in other templates as well.

**Comment by JunTaoLuo** [comment]
Merged in https://github.com/aspnet/Universe/commit/f7c3546b988ae2643b8b0cbdde6cb5cff3dc0e1a

**Pull request closed without merging**